.circleci/config.yml

```diff
@@ -177,6 +177,9 @@ workflows:
       - rtl8720e/loadable_apps:
           requires:
             - checkout_code
-      - rtl8730e/hello:
+      - rtl8730e/hello_8M:
+          requires:
+            - checkout_code
+      - rtl8730e/loadable_apps_8M:
```

Comment:
https://github.com/Samsung/TizenRT/blob/96bbade61ebc84551908bbc4aca0df7eff49c68e/.circleci/config.yml#L138

I think you will have to change the above line too in order to support the newly added config build.

Reply:
done